pom.xml

```diff
@@ -983,6 +991,47 @@
                 <module>assembly</module>
             </modules>
         </profile>
+        <profile>
+            <id>dbaas3.0</id>
+            <activation>
+                <property>
+                    <name>env</name>
+                    <value>dbaas3.0</value>
+                </property>
+            </activation>
+            <properties>
+                <cdh.parcel.version>6.3.2</cdh.parcel.version>
+                <envClassifier>cdh6.3.0</envClassifier>
```

Comment:
We can't reuse _envClassifier_ in this profile. All platform dependent modules (_hbase_sql_, _hbase_storage_, etc.) will be built with the same version from two different profiles: _cdh6.3.0_ and _dbaas3.0_. There will be only one version in Maven repository at any particular time. Which one will depend on which profile was built last.

Reply:
A new subdirectory _assembly/dbaas3.0_ is also needed for the new profile/envClassifier

Reply:
Done